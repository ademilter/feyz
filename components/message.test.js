import { render, screen } from '@testing-library/react'
import Message from './message'

describe('Message', () => {
  test('should have render.', () => {
    const { asFragment } = render(<Message />)

    expect(asFragment()).toMatchSnapshot()
  })
})
