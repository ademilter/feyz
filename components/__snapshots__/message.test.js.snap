// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should have render. 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    <a
      class=""
      href="https://airtable.com/shrWh78cmXzVHcgwV"
      rel="noopener noreferrer"
      target="_blank"
    >
      Kafa açacağını düşündüğün faydalı içerikleri sen de ekleyebilirsin → 
    </a>
  </div>
</DocumentFragment>
`;
